configs/alcf/config.20180410.A_WCYCL1950_HR.ne120_oRRS18v3_ICG.theta

```diff
@@ -111,52 +117,3 @@ endYear = 10
 # For valid statistics, index times should include at least 30 years
 startYear = 1
 endYear = 9999
-
-[oceanObservations]
-## options related to ocean observations with which the results will be compared
-
-# directory where ocean observations are stored
-baseDirectory = /lus/theta-fs0/projects/ClimateEnergy_2/diagnostics/observations/Ocean
-
-[seaIceObservations]
-## options related to sea ice observations with which the results will be
-## compared
-
-# directory where sea ice observations are stored
-baseDirectory = /lus/theta-fs0/projects/ClimateEnergy_2/diagnostics/observations/SeaIce
-
-[timeSeriesSeaIceAreaVol]
-## options related to plotting time series of sea ice area and volume
-
-# plot on polar plot
-polarPlot = False
-
-[regions]
-# Directory containing mask files for ocean basins and ice shelves
-regionMaskDirectory = /lus/theta-fs0/projects/ClimateEnergy_2/diagnostics/mpas_analysis/region_masks
-
```

Comment:
here is another example of section (below this comment) that seems unrelated to this PR and that is being removed. Not sure if it was intentional or not.

Reply:
I really think we don't want this stuff in the example config files, so, yes, it was intentional.